### PR TITLE
Pushing fixes to LUT analysis notebooks

### PR DESCRIPTION
**Pull request by craiglagegit**
I pushed fixes to all three notebooks, and to some code in vandv:

SITCOMTN-079_LUT_analysis_improvement.ipynb
and 
SITCOMTN-079_LUT_analysis_updates.ipynb
Now run start to finish.  I can't judge whether the plots are all correct, but they look reasonable.

SITCOMTN-079_LUT_analysis_measured_forces.ipynb
Has two problems:  
(1)In the 5th cell, it is looking for a function: vandv.m1m3.snapshot_zforces_overview 
that doesn’t exist.
(2)In the 11th cell, the start and end times given to return the df_state data frame returns no data.

I need input on how to fix those.

### CODE REVIEW

**Comment by craiglagegit** [comment]
I think this is done.  All three notebooks now run without errors.  Please review.

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-079_LUT_analysis_improvement.ipynb [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "warnings.filterwarnings('ignore')\n",
```
Comment (gmegh) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-079_LUT_analysis_improvement.ipynb:33]: <pre class="ql-syntax" spellcheck="false">Line #1.    %matplotlib inline
</pre><p>I would remove this line and move the same line in the first cell, at the bottom of the first cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/137/discussion/#comment-1624974476'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (craiglagegit) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-079_LUT_analysis_improvement.ipynb:33]: Done.  Fix pushed.

**Review by b1quint — APPROVED** [review]
It looks good now. Thanks!

**Pull request merged**